website/content/en/configurations/identity-access-modes/service-principal-mode.md

```diff
@@ -70,7 +70,7 @@ spec:
 
 ## Configure Service Principal to access Keyvault
 
-1. Add your service principal credentials as a Kubernetes secrets accessible by the Secrets Store CSI driver. If using AKS you can learn about [service principals in AKS here.](https://docs.microsoft.com/azure/aks/kubernetes-service-principal)
+1. Add your service principal credentials as a Kubernetes secrets accessible by the Secrets Store CSI driver. If using AKS you can learn about [service principals in AKS here.](https://docs.microsoft.com/azure/aks/kubernetes-service-principal) Service Principal is currently the only way to connect to Azure Key Vault from a non Azure environment.

```

Comment:
It looks redundant as we are stating this very clearly at the top of the page, doesn't it?